common/src/main/resources/db/changelog/v2024/ab2d_6151_rename_column.sql

```diff
@@ -0,0 +1,11 @@
+DO $$
+BEGIN
+    IF EXISTS (SELECT 1
+     FROM information_schema.columns
+     WHERE table_schema = 'public' AND table_name = 'current_mbi' AND
+column_name = 'opt_out_flag')
+    THEN
+ALTER TABLE public.current_mbi RENAME opt_out_flag TO share_data;
+END IF;
+END
+$$;
```

Comment:
It also need to be renamed for `update_current_mbi` procedure. May be manually?

Reply:
We dont use "update_current_mbi " anymore remember?  proc_insert_mbi_to_table is what we have the pg_cron set for. that doesnt care for opt_out_flag at all

Reply:
ah, right! Impl still has the good old "update_current_mbi" and this confused me